code/controllers/subsystem/traumas.dm

```diff
@@ -35,6 +35,7 @@ SUBSYSTEM_DEF(traumas)
 		"falling"          = construct_phobia_regex("falling"),
 		"anime"            = construct_phobia_regex("anime"),
 		"insects"  = construct_phobia_regex("insects"),
+		"ocky icky"  = construct_phobia_regex("ocky icky"),
```

Comment:
```suggestion
		"ocky icky" = construct_phobia_regex("ocky icky"),
```

no alignment indentation

Reply:
It's fine if I go and fix all the others too since I'm here right? I wanted to ask this earlier but forgot.